scipy/spatial/transform/tests/test_rotation.py

```diff
@@ -619,3 +619,86 @@ def test_inv_single_rotation():
 
     assert_array_almost_equal(result1, eye3d)
     assert_array_almost_equal(result2, eye3d)
+
+
+def test_apply_single_rotation_single_point():
+    r_1d = Rotation.from_dcm([
+        [0, -1, 0],
+        [1, 0, 0],
+        [0, 0, 1]
+    ])
+    r_2d = Rotation.from_dcm([[
```

Comment:
Can we use `np.atleast_3d` or something?

Reply:
So `np.atleast_3d` adds dimensions to the end. It takes us from `(3, 3) -> (3, 3, 1)` whereas we require `(3, 3) -> (1, 3, 3)`.
The [documentation](https://docs.scipy.org/doc/numpy-1.14.0/reference/generated/numpy.atleast_3d.html) for this function does not show a way to achieve this so I'm using `np.expand_dims` instead.